build_RRFS-CMAQ.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+############################################################
+## Script to build RRFS-CMAQ                              ##
+## Components:                                            ##
+## - regional workflow, UFS_UTILS, UPP                    ##
+## - arl_nexus, fv3gfs_aqm, gefs2clbcs, upp_post_stat     ##
+## - GSI, JEDI (only for DA)                              ##
+############################################################
+
+set -eu
+
+if [[ $(uname -s) == Darwin ]]; then
+  readonly MYDIR=$(cd "$(dirname "$(greadlink -f -n "${BASH_SOURCE[0]}" )" )" && pwd -P)
+else
+  readonly MYDIR=$(cd "$(dirname "$(readlink -f -n "${BASH_SOURCE[0]}" )" )" && pwd -P)
+fi
+
+###########################################################
+## User specific parameters                              ##
+###########################################################
+##
+## DA option ("NO" or "YES")
+##    NO: hera, wcoss_dell_p3
+##    YES: hera
+##
+DA_opt="NO"
```

Comment:
@chan-hoo You can make `DA_opt` an optional argument so you don't have to change the script to reset it. e.g.:
`
DA_opt=${$1:-NO}
`

Reply:
@gsketefian, good idea! Modified.